mesonbuild/compilers/compilers.py

```diff
@@ -127,6 +129,13 @@ def is_library(fname):
                           'release': ['-O3'],
                           'minsize': ['-Os', '-g']}
 
+cuda_buildtype_args = {'plain': [],
+                       'debug': ['-O0', '-g'],
+                       'debugoptimized': ['-O1', '--debug'],
+                       'release': ['-O3', '-Otime'],
+                       'minsize': ['-O3', '-Ospace'],
+                      }
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [SideCI](https://sideci.com) -->